account/manager.go

```diff
@@ -165,10 +166,44 @@ func actionTxLabel(account *Account, action Action, isExpirySpend bool,
 	if err != nil {
 		log.Errorf("Internal error: failed to serialize json "+
 			"from %v: %v", label, err)
-		return fmt.Sprintf(prefixTag, action)
+		return fmt.Sprintf(txLabelPrefixTag, action)
+	}
+
+	return fmt.Sprintf(txLabelPrefixTag, labelJson)
+}
+
+// IsPoolTx returns true if the given transaction is related to
+// pool.
+func IsPoolTx(tx *lnrpc.Transaction) bool {
+	return strings.HasPrefix(tx.Label, txLabelPrefixTag)
```

Comment:
This doesn't work, because of the `%s`. Maybe we should remove the formatting part and just use that where we actually write a string? Otherwise we'd have to add the `prefix := fmt.Sprintf(txLabelPrefixTag, "")` here too.

Reply:
I will remove `%s` from the tag, simpler to keep formatting symbols with formatting function calls.